### ISSUE TITLE
Create a new maven release with more precise time stamps for comments

### ISSUE THREAD

**Issue by tdauth** [rcarz/jira-client#288]
In my dependency 0.5 it still uses SimpleDateFormat with yyyy-MM-dd. I see that it has been improved at the current master branch:
- https://github.com/rcarz/jira-client/blob/master/src/main/java/net/rcarz/jiraclient/Comment.java#L67
- https://github.com/rcarz/jira-client/blob/master/src/main/java/net/rcarz/jiraclient/Field.java#L259
- https://github.com/rcarz/jira-client/blob/master/src/main/java/net/rcarz/jiraclient/Field.java#L155

Why is there no maven release like 0.6?

**Comment by pietrygamat** [comment]
I would not get my hopes up waiting for new release here - there was no merge to master in years and no reply from the maintainer to any of the host of questions about 0.6 release posted regularly [since 2015](https://github.com/rcarz/jira-client/issues/81). 

I am only replying in this thread because I get notified about new questions like this every now and then, ever since I thumbed up one of the first in the looong series.

Quick search through [maven central](https://search.maven.org/search?q=jira-client) and through recent activity in [this project github network]( https://github.com/rcarz/jira-client/network) reveals that there is at least [one fork](https://github.com/nexocodecom/jira-client) of this repository that had public release within last 18 months and [another fork](https://github.com/andreas-wenzel/de-bwl-lgl-jira-client) that has recent repository activity. 

I am not representing or even using either project, and know nothing about their maintainers' willingness to commit, just pointing the alternatives. Of course, one cannot ignore the fact that since there seems to be little interest in picking up work after rcarz, maybe the official api by Atlassian got equaly user friendly after all those years, and it's the one that everyone migrated to? 

Best of luck

**Comment by tdauth** [comment]
Yes I am thinking about using the Atlassian API or something else.
